_build/stac.hbs

```diff
@@ -48,7 +48,9 @@
 {{/each}}
     {
       "href": "{{{escapeJson LicenseUrl}}}",
-      "rel": "license"
+      "rel": "license",
+      "title": "License",
+      "type": "text/html"

```

Comment:
Should this be mimeType or licence type? In yaml files sometimes there is `LicenseType` key with value `proprietary` which makes me believe this shouldn't be mime type

Reply:
This gets transformed to an object inside of the `links` array in stac json files where `type` tells the consumer (another script or a human) what mime type / format the link at that URL is. So, I think it should be mime type.